mlir/utils/performance/ck-benchmark-driver/ck-benchmark-driver.cpp

```diff
@@ -12,6 +12,8 @@
 //
 //===----------------------------------------------------------------------===//
 
+#ifdef CK_FOUND
```

Comment:
Im bumping into :
`clang-tidy found error: mlir/utils/performance/ck-benchmark-driver/ck-benchmark-driver.cpp:19:10: error: 'ck/ck.hpp' file not found [clang-diagnostic-error]`
http://ml-ci-internal.amd.com:8080/blue/organizations/jenkins/MLIR%2Fmlir/detail/PR-1404/8/pipeline/186

@giuseros I had to do this to avoid clang-tidy erroring out when ck headers are not found.
I am not quite sure how this ever worked coz looking at the jenkins pipeline we run get and build ck way later -- so the header is not going to be there. Maybe Im missing something?